core/primitives/src/views.rs

```diff
@@ -322,6 +322,33 @@ pub struct ValidatorInfo {
     pub is_slashed: bool,
 }
 
+#[cfg_attr(feature = "deepsize_feature", derive(deepsize::DeepSizeOf))]
+#[derive(Serialize, Deserialize, Debug)]
+pub struct DebugChunkStatus {
+    pub shard_id: u64,
+    pub chunk_hash: ChunkHash,
+    pub gas_used: u64,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub processing_time_ms: Option<u64>,
+}
+
+#[cfg_attr(feature = "deepsize_feature", derive(deepsize::DeepSizeOf))]
+#[derive(Serialize, Deserialize, Debug)]
+pub struct DebugBlockStatus {
+    pub block_hash: CryptoHash,
+    pub block_height: u64,
+    pub chunks: Vec<DebugChunkStatus>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub processing_time_ms: Option<u64>,
```

Comment:
comment the fields `processing_time_ms` and `timestamp_delta`, what they mean, how they are calculated

Reply:
Done.